ivy/container/elementwise.py

```diff
@@ -6473,3 +6473,199 @@ def minimum(
 
         """
         return self.static_minimum(self, x2, key_chains, to_apply, prune_unapplied, map_sequences, out=out)
+
+
+
+    @staticmethod
+    def static_maximum(
+        x1: Union[ivy.Container, ivy.Array, ivy.NativeArray],
+        x2: Union[ivy.Container, ivy.Array, ivy.NativeArray],
+        key_chains: Optional[Union[List[str], Dict[str, str]]] = None,
+        to_apply: bool = True,
+        prune_unapplied: bool = False,
+        map_sequences: bool = False,
+        *,
+        out: Optional[ivy.Container] = None,
+    ) -> ivy.Container:
+        """Returns the max of x1 and x2 (i.e. x1 > x2 ? x1 : x2) element-wise.
```

Comment:
this docstring is wrong, check out the **classes** section [here](https://lets-unify.ai/ivy/deep_dive/12_docstrings.html) for details.